source/documentation/device_wifi.md

```diff
@@ -0,0 +1,28 @@
+# Certificate Authentication (Alpha)
+
+## What is Certificate Authentication
+Device-Wifi requires a user to have either a government email address, be sponsored by someone with a government email address, or have a mobile phone number. Device-Wifi only requires that a certificate needs to be installed on a device to gain access, this removes the need to always enter an email and password to gain access to government Wifi.
```

Comment:
Should read "GovWifi requires..."

Reply:
I think this could be better explained by the need, which is organisations needing to connect many devices to the internet before knowing the users' username and password. Certificate based authentication means that a device can connect rather than a person, which solves the problem of an IT manager needing to connect dozens of managed devices, often at once.

Reply:
Could also mention that we have noticed that network engineers sometimes re-use one of their own personal username and password multiple times for managed devices as a workaround for not having a certificate based system to use.